tests/pages/firefox/welcome/page4.py

```diff
@@ -0,0 +1,44 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from pages.firefox.base import FirefoxBasePage
+from pages.regions.modal import Modal
```

Comment:
Add `ModalProtocol`. I'm not sure why we need to import both, but it seems we do.

`from pages.regions.modal import Modal, ModalProtocol`